extension/war/src/main/webapp/groovy/social/webui/profile/UIRecentActivity.gtmpl

```diff
@@ -14,13 +15,28 @@ if(activity == null) {
 
 <div id="${uicomponent.id}" class="uiRecentActivity">
 <%
+  String FILE_SPACES = "files:spaces";
+  String DEFAULT_ACTIVITY = "DEFAULT_ACTIVITY";
+  String NEWS = "news";
   String activityURL = LinkProvider.getBaseUri(null, null) + "/activity?id=";
   def profile = RecentActivitiesHelper.getOwnerActivityProfile(activity);
   String avatarURL = profile.getAvatarUrl();
   String profileURL = profile.getUrl();
   String displayName = StringEscapeUtils.escapeHtml(profile.getFullName());
   String link = RecentActivitiesHelper.getLink(activity);
   String linkTitle = RecentActivitiesHelper.getLinkTitle(activity);
+  String activityTitle = "";
+  if (DEFAULT_ACTIVITY.equals(activity.getType())){
+     activityTitle = activity.getTitle();
+  }else if(NEWS.equals(activity.getType())){
+     activityTitle = _ctx.appRes("ProfileUpdatesPublisher.news.uploaded");
+  } else if (FILE_SPACES.equals(activity.getType())) {
+     String fileName = activity.getTemplateParams().get("contentName");
+     if (StringUtils.isNotBlank(fileName)){
+       fileName = uicomponent.CleanName(fileName);
```

Comment:
```suggestion
       fileName = uicomponent.decodeFileName(fileName);
```

Reply:
This method to return a clean name, decode the fileName in case the fileName is already encoded, for the old uploaded files if not will return the same name in case the name is already decoded